.github/workflows/homebrew_publish.yml

```diff
@@ -0,0 +1,47 @@
+# Github action to open a PR in Homebrew/homebrew-core when a new release of ballerina is being published
+name: Update Homebrew
+# on:
+#   release:
+#     types: [published]
+
+on:
+  push:
+    branches:
+      - prod-sync
+    paths:
+      - '_data/latest/**'
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository.
+        uses: actions/checkout@v2
+      
+      - name: Set up Python 3.x
+        uses: actions/setup-python@v1
+        with:
+          python-version: '3.x'
+        
+      - name: Wget required files
+        id: version
+        run: wget -O ./.github/scripts/latest_release.json https://product-dist.ballerina.io/downloads/latest_release.json?982
+        
+      - name: Send the PR to Homebrew
+        run: |
+          python -m pip install --upgrade pip
+          pip install PyGithub
+          version=$(cat ./.github/scripts/latest_release.json | jq '.version')
```

Comment:
Shall we get the latest version as below.

https://github.com/ballerina-platform/ballerina-dev-website/blob/master/.github/workflows/add_new_release_note.yml#L25

Also, just be on the safe side. Let's install jq as below.
https://github.com/ballerina-platform/ballerina-dev-website/blob/master/.github/workflows/add_new_release_note.yml#L17